test/test-MSONMixinParser.cc

```diff
@@ -41,7 +41,7 @@ TEST_CASE("Parse canonical mson mixin", "[mson][mixin]")
     NamedTypeHelper::build("Person", mson::ObjectBaseType, namedTypes);
 
     ParseResult<mson::Mixin> mixin;
-    SectionParserHelper<mson::Mixin, MSONMixinParser>::parseMSON(source, MSONMixinSectionType, mixin, 0, namedTypes);
+    SectionParserHelper<mson::Mixin, MSONMixinParser>::parseMSON(source, MSONMixinSectionType, mixin, ExportSourcemapOption, namedTypes);
```

Comment:
Now one could argue that one unit is parsing the MSON and another unit is producing the source map and those two should be tested independently of each another and rely on each other. Please keep this in mind next time and make a separate tests for parsing unit and source map units, it will help for example when you are refactoring just one part of it...